clap_derive/Cargo.toml

```diff
@@ -44,6 +44,8 @@ proc-macro-error = "1"
 [features]
 default = []
 debug = []
+unstable-v4 = ["unstable-array"]
```

Comment:
I've split this out into #3702 as some changes I'm working on right now are blocked on it